### PR TITLE
set clustering defaults

### PR DESCRIPTION
**Pull request by martacki**
Closes # (if applicable).
#412

## Changes proposed in this Pull Request
in the same time, move from `numpy` aggregation to `pandas` aggregation

edit: commit `b3437b2` is already subject to a different PR, I forgot to remove it here, but it shouldn't really matter.

## Checklist
- [ ] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [ ] Unit tests for new features were added (if applicable).
- [x] Newly introduced dependencies are added to `environment.yaml`, `environment_docs.yaml` and `setup.py` (if applicable).
- [ ] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [x] I consent to the release of this PR's code under the MIT license.

### CODE REVIEW

**Comment by p-glaum** [comment]
Maybe in the future, we could change the aggregation strategy of the generator lifetime to mean instead of np.inf.

**Comment by martacki** [comment]
@fneum we're not sure why build `3.7 windows-latest` isn't starting, no other review is required (unless you want to review of course! ;))

@p-glaum you can submit your review, i.e. comment/approve/request changes in the upper right field `files changed` -> `review changes`.

File: pypsa/networkclustering.py [hunk 1]
```diff
@@ -91,16 +91,23 @@ def normed_or_uniform(x):
...
+        "weight": pd.Series.sum,
+        "p_nom": pd.Series.min,
+        "capital_cost": pd.Series.sum,
+        "efficiency": pd.Series.mean,
+        "ramp_limit_up": pd.Series.mean,
+        "ramp_limit_down": pd.Series.mean,
+        "ramp_limit_start_up": pd.Series.mean,
+        "ramp_limit_shut_down": pd.Series.mean,
+        "build_year": lambda x: 0,
+        "lifetime": lambda x: np.inf,
```
Comment (p-glaum) [inline pypsa/networkclustering.py:106]: We should check in the future if these default values are suited.

**Review by fneum — APPROVED** [review]

**Comment by martacki** [comment]
correct. I will fix it, thanks @pz-max @coroa

**Pull request merged**